app/controllers/service_provider_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ServiceProviderController < ApplicationController
+  protect_from_forgery with: :null_session
+
+  def update
+    # TODO: shared secret token to limit authz to corresponding env dashboard
+
+    if FeatureManagement.use_dashboard_service_providers?
+      ServiceProviderUpdater.new.run
+      SecureHeadersWhitelister.new.run
+    end
+
+    # always return success in case feature is off
+    render json: { status: 'thanks' }
```

Comment:
maybe something similar to the code comment above? Like "If the dashboard service provider feature is turned on, update was successful" ? Sounds awkward, but it is giving info!